src/Features/OnboardingTasks/Tasks/Purchase.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Automattic\WooCommerce\Admin\Features\OnboardingTasks\Tasks;
+
+use Automattic\WooCommerce\Admin\Features\Onboarding;
+use Automattic\WooCommerce\Admin\PluginsHelper;
+
+/**
+ * Purchase Task
+ */
+class Purchase {
+	/**
+	 * Get the task arguments.
+	 *
+	 * @return array
+	 */
+	public static function get_task() {
```

Comment:
Since some of these tasks have admin scripts, I'm considering making them self-registering.  Might handle in a follow-up since admin scripts have not yet been moved if we decide it's a good idea.

```php
// TaskLists.php
new Purchase();

// Purchase.php
__construct() {
  TaskLists::add_task( 'setup', array( __CLASS__, 'get_task' );
  // Register any scripts needed for the task
}
```